src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -43,6 +68,9 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
                 _ => throw new ArgumentException()
             };
 
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                engineBinary = GetFullPathTo(engineBinary);
```

Comment:
This should probably not do this if the path is already rooted

Reply:
The `engineBinary` is set few lines above and is not rooted. So hopefully we don't need that?

Reply:
hah! ignore me!

Reply:
OTOH, it is easy to do, so I will handle it just in case.